assets/models/system/automl-image-classification/asset.yaml

```
@@ -1,4 +1,4 @@
 extra_config: model.yaml
 spec: spec.yaml
 type: model
-categories: ["Models"]
+categories: ["Foundation Models", "Automl"]
```

```suggestion
categories: ["Foundation Models", "AutoML"]
```

Same at other places as well.

---

done